explainaboard/metric.py

```diff
@@ -56,13 +57,13 @@ def __len__(self) -> int:
         """
         Returns the number of samples in the dataset
         """
-        return len(self._data)
```

Comment:
This is actually a bug, fixed in https://github.com/neulab/ExplainaBoard/pull/188

Reply:
Keeps the change as-is for avoiding the typing error. This should be resolved at merging #188, or separate pr to resolve this specific problem is needed.